### PR TITLE
fix:'hidden' attr is not work in tag element

### PR DESCRIPTION
**Pull request by bainian**

### CODE REVIEW

**Comment by bainian** [comment]
fix:  #97

File: webf/lib/src/dom/element.dart [hunk 1]
```diff
@@ -1002,6 +1002,11 @@ abstract class Element extends Node with ElementBase, ElementEventMixin, Element
 
   @mustCallSuper
   void setAttribute(String qualifiedName, String value) {
+    if (value.startsWith(';')) value = value.substring(1);
```
Comment (andycall) [inline webf/lib/src/dom/element.dart:1005]: What's the purpos of this line?

File: webf/lib/src/dom/element.dart [hunk 2]
```diff
@@ -1002,6 +1002,11 @@ abstract class Element extends Node with ElementBase, ElementEventMixin, Element
...
+    if (HIDDEN == qualifiedName) {
```
Comment (andycall) [inline webf/lib/src/dom/element.dart:1006]: Hidden attribute have three different kind of values. 

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden

image

**Review by andycall — CHANGES_REQUESTED** [review]

File: webf/lib/src/dom/element.dart (hunk 2 above)
Comment (bainian) [reply webf/lib/src/dom/element.dart:1006]: thx,  it really not comprehensive, I try to handle it.

File: webf/lib/src/dom/element.dart (hunk 1 above)
Comment (bainian) [reply webf/lib/src/dom/element.dart:1005]: UICommand pass some attr values start with a semicolon, it seems  it can be ignore handle here .

Comment (andycall) [reply webf/lib/src/dom/element.dart:1005]: It should't handled here, it's an base road for setAttribute API. Please identify the source of property and value which contains the semicolon and fixed for that property.

**Comment by andycall** [comment]
test specs are missing.

**Pull request closed without merging**